react/features/base/lib-jitsi-meet/index.js

```diff
@@ -6,6 +6,8 @@ export { JitsiMeetJS as default };
 // XXX Re-export the types exported by JitsiMeetJS in order to prevent undefined
 // imported JitsiMeetJS. It may be caused by import cycles but I have not
 // confirmed the theory.
```

Comment:
The comment is now in the wrong place.

Reply:
Why?

Reply:
The comment is about types and was placed above variables that hold types. Now the comment is above analytics and a browser type util.

Reply:
Hmm... I think what the comment is describing is that we are exporting variables in the global scope for the file which will be problematic if there is circular dependancies. But this includes any type of variables - objects, function, class, etc. Isn't this correct?

Maybe types is now a little confusing after the addition of analytics and browser type. What about replacing types with variables? Or properties maybe?